### PR TITLE
Update publications.yml

### PR DESCRIPTION
**Pull request by romain-lopez**
added NeurIPS paper

### CODE REVIEW

File: _data/publications.yml [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+  details: Advanced in Neural Information Processing Systems
+  contentshare: https://github.com/PierreBoyeau/sbVAE
+  arxiv: https://arxiv.org/abs/2002.07217
+
 - authors: "R. Lopez, A. Gayoso & N. Yosef."
   title: "Enhancing Scientific Discoveries in Molecular Biology with Deep Generative Models."
   year: 2020
   doi: 10.15252/msb.20199198
   details: Molecular Systems Biology
+  file: https://www.embopress.org/doi/10.15252/msb.20199198
```
Comment (adamgayoso) [inline _data/publications.yml]: the doi provides the hyperlink already

File: _data/publications.yml [hunk 2]
```diff
@@ -1,11 +1,19 @@
 # THE WEIRDEST BUG: using escapes requires single-quotes instead of double...
 # filetype options: "pdf - published version" "pdf - author manuscript version (peer-reviewed & edited)"...
 
+- authors: "R. Lopez, P. Boyeau, N. Yosef, M. Jordan & J. Regier."
+  title: "Decision-Making with Auto-Encoding Variational Bayes."
+  year: 2020
+  details: Advanced in Neural Information Processing Systems
```
Comment (adamgayoso) [inline _data/publications.yml]: advances?

File: _data/publications.yml (hunk 1 above)
Comment (adamgayoso) [reply _data/publications.yml]: so this isn't necessary

**Review by adamgayoso — CHANGES_REQUESTED** [review]

**Comment by romain-lopez** [comment]
Tx

**Pull request merged**